ERCS/erc-7674.md

```diff
@@ -0,0 +1,58 @@
+---
+eip: 7674
+title: Temporary Approval Extension for ERC-20
+description: An interface for ERC-20 approvals via transient storage
+author: Xenia Shape (@byshape), Mikhail Melnik (@ZumZoom), Hadrien Croubois (@Amxx)
+discussions-to: https://ethereum-magicians.org/t/add-erc-7674-transient-approval-extension-for-erc-20/19521
+status: Draft
+type: Standards Track
+category: ERC
+created: 2024-04-02
+requires: 20, 1153
+---
+
+## Abstract
+
+This specification defines the minimum interface required to temporarily approve `ERC-20` tokens for spending within a single transaction.
```

Comment:
same comment: "single block" is incorrect.

Reply:
Yes. Updated accordingly

Reply:
I actually prefer this wording instead of adding extra text in parens:

```suggestion
This specification defines the minimum interface required to temporarily approve `ERC-20` tokens for spending within the same transaction.
```